src/enums/accounts.js

```diff
@@ -29,7 +29,9 @@ const LedgerAccountType = {
   SETTLEMENT: 2,
   HUB_RECONCILIATION: 3,
   HUB_MULTILATERAL_SETTLEMENT: 4,
-  HUB_FEE: 5
+  HUB_FEE: 5,
+  POSITION_REMITTANCE: 7,
```

Comment:
@vijayg10 see slack message with regard to this.

Reply:
Hi @mdebarros, can you re-review based on our conversation in slack?